apps/expo/package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "@deenup/expo",
 	"version": "0.1.0",
 	"private": true,
-	"main": "expo-router/entry",
+	"main": "index.ts",
```

Comment:
why was this changed?

Reply:
The build is not able to resolve an entry file so I had to create one and put it there